backend/src/controllers/items-controller.ts

```diff
@@ -125,8 +136,18 @@ export const updateItem = async (req: Request, res: Response) => {
 }
 
 //Deletes a specific item
-export const deleteItem = async (req: Request, res: Response) => {
+export const deleteItem = async (req: CustomRequest, res: Response) => {
     const itemID: string = req.params._id;
+    const ownerId = req.body.itemOwnerId;
```

Comment:
Similar to above: Remove the `ownerId`, and replace the filter below with `req.token.userId`